app/decorators/school_decorator.rb

```diff
@@ -3,6 +3,13 @@ class SchoolDecorator < OrganisationDecorator
   attribute :walk_travel_duration
   attribute :drive_travel_duration
 
+  def formatted_duration(mode)
+    duration = send("#{mode}_travel_duration")
+    return "" if duration.blank?
+
+    duration.gsub(/\bmins\b/, "minutes")
```

Comment:
@Kizr alluded to a way a way of doing this with translations.
I would at least suggest moving "minutes" to a locale.

Reply:
"minutes" has been added to a locale.